mmv1/provider/terraform/examples.rb

```diff
@@ -87,6 +87,14 @@ class Examples < Api::Object
       # See test_vars_overrides for more details
       attr_reader :oics_vars_overrides
 
+      # Hash to provider custom override values for generating tf_cloud_docs
+      # configs. See test_vars_overrides for more details
+      attr_reader :cloud_docs_vars_overrides
+
+      # The region tag that wraps the primary resource as a snippet in a
+      # cloud docs sample.
+      attr_reader :cloud_docs_region_tag
```

Comment:
Thoughts on these override names? I considered `region_tag` over `cloud_docs_region_tag`, but I wanted to be consistent with `cloud_docs_vars_overrides`

Reply:
Yeah, consistency sgtm